Library/CreditPayment.cs

```diff
@@ -10,7 +10,7 @@ public class CreditPayment : RecurlyEntity
     {
         public string Uuid { get; set; }
         public string Action { get; set; }
-        public int UnitAmountInCents { get; set; }
+        public int AmountInCents { get; set; }
```

Comment:
This isn't a breaking change as the old attribute was incorrect, right?

Reply:
Hmmm. Well the old attribute was incorrect in that it would always return a value of 0. I'm not sure if this counts as a bug fix or a breaking change, in that yes, someone's code will break if they try to parse `UnitAmountInCents` (they will encounter `error CS1061: 'CreditPayment' does not contain a definition for 'UnitAmountInCents'`). If we treat this as a breaking change, I think I ought to change the base branch for this PR.

Reply:
> code will break if they try to parse

Maybe I have a misunderstanding. Is this strictly for parsing or did it expose a public interface i.e. `creditPayment.UnitAmountInCents`?